packages/mobile/src/components/lineup-tile/LineupTileStats.tsx

```diff
@@ -42,33 +43,38 @@ const formatPlayCount = (playCount?: number) => {
 }
 
 const useStyles = makeStyles(({ spacing, palette }) => ({
-  stats: {
+  root: {
     flexDirection: 'row',
     alignItems: 'stretch',
     paddingVertical: spacing(2),
-    marginHorizontal: spacing(2.5)
+    marginHorizontal: spacing(2.5),
```

Comment:
I don't see spacing by half units much. Is this part of the design?
I guess track tile is from before the standardization?

Reply:
yeah I've noticed this as well - actually the new designs have `spacing(2)` borders but I don't wanna make that large of a change in this PR. But this needs to be fixed! Believe I've alerted @dylanjeffers about it before but will call out again.